test/main_dummy_controller_test.cpp

```diff
@@ -1,54 +1,37 @@
-#include "hwlib.hpp"
+#include "ostream"
```

Comment:
This must be between <>

Reply:
This is fixed in a later commit (from a while ago, so i can't find the commit, but you can see it changed in release)